vmdb/app/models/miq_expression.rb

```diff
@@ -61,6 +63,12 @@ class MiqExpression
     cloud_tenants
     compliances
     compliance_details
+    configuration_profiles
+    configuration_managers
+    configured_systems
+    customization_scripts
+    customization_script_ptable
+    customization_script_medium
```

Comment:
customization_script_ptable and customization_script_medium need to be pluralized.
```
irb(main):002:0> "customization_script_medium".pluralize
=> "customization_script_media"
```